src/core/labeling/qgslabelingengine.h

```diff
@@ -29,12 +27,30 @@ class QgsLabelingResults;
 class QgsLabelFeature;
 class QgsLabelingEngineSettings;
 
+#ifndef SIP_RUN
 namespace pal
 {
   class Problem;
   class Pal;
   class LabelPosition;
 }
+#endif
+
+/**
+ * \ingroup core
+ * \brief Represents a label candidate.
+ */
+class CORE_EXPORT QgsLabelCandidate
```

Comment:
I'm always wondering if it wouldn't be worth mentioning a since for the Python API?

Reply:
This was already exposed to sip, its just moved to a different header file now